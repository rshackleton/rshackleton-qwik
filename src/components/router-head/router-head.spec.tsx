import { QwikCityMockProvider } from '@builder.io/qwik-city';
import { createDOM } from '@builder.io/qwik/testing';
import { expect, test } from 'vitest';
import { RouterHead } from './router-head';

test(`[Router Head Component]: Renders without error`, async () => {
  const { screen, render } = await createDOM();
  await render(
    <QwikCityMockProvider>
      <RouterHead />
    </QwikCityMockProvider>,
  );
  expect(screen.outerHTML).toMatchSnapshot();
});
