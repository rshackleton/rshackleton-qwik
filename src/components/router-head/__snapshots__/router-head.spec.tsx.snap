// Vitest Snapshot v1

exports[`[Router Head Component]: Renders without error 1`] = `"<host q:version=\\"0.16.2\\" q:container=\\"resumed\\" q:render=\\"dom-dev\\"><!--qv q:key=WmYC:vl_1 data-qrl=QwikCityMockProvider_component_WmYC5H00wtI--><!--qv q:key q:sref=0 q:s--><!--qv q:key=zrbr:vl_0 data-qrl=RouterHead_component_zrbrqoaqXSY--><title></title><link rel=\\"canonical\\" href=\\"http://localhost/\\"><meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\"><link rel=\\"icon\\" type=\\"image/svg+xml\\" href=\\"/favicon.svg\\"><!--/qv--><!--/qv--><!--/qv--></host>"`;
